src/components/common/SettingsHeader.js

```diff
@@ -32,7 +32,7 @@ const useStyles = makeStyles((theme) => ({
   },
   username: {
     margin: theme.spacing(0, 2),
-    maxWidth: 100,
+    maxWidth: 120,
```

Comment:
It's a detail, but could you factor out this number as `HEADER_USERNAME_MAX_WIDTH` and define it in `constants` right below `USERNAME_MAX_LENGTH`? Then it will be ready to merge!